mne/preprocessing/_csd.py

```diff
@@ -175,3 +177,123 @@ def compute_current_source_density(inst, sphere='auto', lambda2=1e-5,
+
+    First an electrical distance matrix is computed by taking the pairwise
+    variance. Then, a local maximum near 0 :math:`\mu`V:sup:`2` and a
+    a local minimum below 5 :math:`\mu`V:sup:`2` are found, the presence
+    of which is indicative of bridging. Finally, electrode distances below
+    the local minimum are marked as bridged as long as they happen on more
+    than the ``epoch_threshold`` proportion of epochs.
+
+    Based on :footcite:`TenkeKayser2001,GreischarEtAl2004,DelormeMakeig2004`
+    and the `EEGLAB implementation
+    <https://psychophysiology.cpmc.columbia.edu/software/eBridge/index.html>`_.
+
+    Parameters
+    ----------
+    inst : instance of Raw, Epochs or Evoked
+        The data to compute electrode bridging on.
+    lm_cutoff : float
+        The distance in :math:`{\\mu}`V:sup:`2` cutoff below which to
+        search for a local minimum (lm) indicative of bridging.
+        Defaults to 16 :math:`{\\mu}`V:sup:`2` to be conservative based
+        on the distributions in :footcite:`GreischarEtAl2004`.
+    epoch_threshold : float
+        The proportion of epochs with electrical distance less than
+        ``ed_threshold`` in order to consider the channel bridged.
+        The default is 0.5.
+    l_freq : float
+        The low cutoff frequency to use. Default is 0.5 Hz.
+    h_freq : float
+        The high cutoff frequency to use. Default is 30 Hz.
+    epoch_duration : float
+        The time in seconds to divide the raw into fixed-length epochs
+        to check for consistent bridging. Only used if ``inst`` is
+        :class:`mne.io.BaseRaw`. The default is 2 seconds.
+    bw_method : None
+        ``bw_method`` to pass to :class:`scipy.stats.gaussian_kde`.
+    %(verbose)s
+
+    Returns
+    -------
+    bridged_idx : list of tuple
+        The indices of channels marked as bridged with each bridged
+        pair stored as a tuple.
+    ed_matrix : ndarray of float, shape (n_channels, n_channels)
+        The electrical distance matrix for each pair of EEG electrodes.
+
+    Notes
+    -----
+    .. versionadded:: 1.1
+
+    References
+    ----------
+    .. footbibliography::
+    """
+    from scipy.stats import gaussian_kde
+    from scipy.optimize import minimize_scalar
+    _check_preload(inst, 'Computing bridged electrodes')
+    inst = inst.copy()  # don't modify original
+    picks = pick_types(inst.info, eeg=True)
+    if len(picks) == 0:
+        raise RuntimeError('No EEG channels found, cannot compute '
+                           'electrode bridging')
+    # first, filter
+    inst.filter(l_freq=l_freq, h_freq=h_freq, picks=picks, verbose=False)
+
+    if isinstance(inst, BaseRaw):
+        inst = make_fixed_length_epochs(inst, duration=epoch_duration,
+                                        preload=True, verbose=False)
+
+    # standardize shape
+    data = inst.get_data(picks=picks)
+    if isinstance(inst, Evoked):
+        data = data.reshape((1,) + data.shape)  # expand evoked
```

Comment:
```suggestion
        data = data[np.newaxis, ...]  # expand evoked
```
if for me more readable